build-tools/packages/build-tools/src/fluidBuild/tasks/leaf/miscTasks.ts

```diff
@@ -14,12 +14,18 @@ import { LeafTask, LeafWithDoneFileTask, LeafWithFileStatDoneFileTask } from "./
 const { verbose } = defaultLogger;
 
 export class EchoTask extends LeafTask {
+	protected get taskWeight() {
+		return 0; // generally cheap relative to other tasks
```

Comment:
Worth declaring some constants for common weights?

Reply:
`LeafTask.taskWeight` is the default (1)